README.md

```diff
@@ -12,67 +13,39 @@ Please see the [INSTALL.md](INSTALL.md).
 
 ![Architecture diagram](SD_Qubes_CI.png)
```

Comment:
Tiny diagram nit: `for for https://ws-ci...` under step 9.

Reply:
Thanks! Fixed.